### ISSUE TITLE
Kite cannot detect Pycharm Community Edition 2020.2.3-4 even when it is running

### ISSUE THREAD

**Issue by elter-tef** [kiteco/issue-tracker#535]
I use Arch Linux (Linux 5.9.14-arch1-1).
I use Pycharm Community Edition 2020.2.3-4. I install it using [arch package](https://archlinux.org/packages/community/x86_64/pycharm-community-edition/).
Pycharm Community Edition is installed in /usr/share/pycharm/bin/pycharm.sh
The process name is /bin/sh /usr/share/pycharm/bin/pycharm. sh

**Comment by cocoonkid** [comment]
I have the same Issue. It works for vim and vscode tho.

A fix would really be highly appreciated!!

**Comment by RigoLigoRLC** [comment]
Pycharm CE 2021.1, Arch package, not detected. Even after installing the plugin from the IDE, it still doesn't work.
![image](https://user-images.githubusercontent.com/49363666/116787870-33495080-aad9-11eb-9e74-8dd71250326c.png)

**Comment by tracure1337** [comment]
I found that installing pycharm via the Jetbrains toolbox (also available as arch AUR package) works.
This way it is installed in the proper path that kite also uses to determine the installed editors.

It's a workaround and it still would be helpful to be able to specify the path in kite editor options itself.

**Comment by RigoLigoRLC** [comment]
Yes, I know this because the documentation said that Kite will look for
editors in ~/.local/JetBrains/toolbox so I installed my IDEs with toolbox
after reporting the issue. But I still hope this problem can be fixed.
